core/bug_api.php

```diff
@@ -544,6 +546,52 @@ function create() {
 		history_log_event_direct( $this->id, 'status', $t_original_status, $t_status );
 		history_log_event_direct( $this->id, 'handler_id', 0, $this->handler_id );
 
+		# Now that the issue is added process the @ mentions
+		$t_all_mentioned_user_ids = array();
```

Comment:
I've just pushed the change to separate mention processing from issue/note creation API.  Please review.